js_modules/dagit/src/configeditor/ConfigEditorUtils.tsx

```diff
@@ -131,40 +131,50 @@ export function scaffoldConfig(pipeline: ConfigEditorPipelineFragment): string {
     Path: "/path/to/file",
     String: "value",
     Int: 1,
-    Boolean: "false"
+    Bool: false,
+    Float: 1.0
   };
 
-  const configPlaceholderFor = (
-    typeName: string,
-    commentDepth: number
-  ): any => {
-    if (placeholders[typeName]) {
-      return placeholders[typeName];
+  const configPlaceholderFor = (typeKey: string, commentDepth: number): any => {
+    if (placeholders[typeKey] !== undefined) {
+      return placeholders[typeKey];
     }
 
-    const type = pipeline.configTypes.find(t => t.name === typeName);
-    if (!type || type.__typename !== "CompositeConfigType") return null;
+    if (typeKey.startsWith("List.")) {
+      const innerPlaceholder = configPlaceholderFor(
```

Comment:
Ah I see. Can you create a task to define a GraphQL interface for what you want here in terms of accessing the inner type names without having to do substring chicanery?